### ISSUE TITLE
Mouse tracking/echo reads incorrectly in the list of recent activity in the Microsoft Store

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8202]
### Steps to reproduce:
1. Start NVDA.
2. Enable the "mouse tracking" function of NVDA from the "Mouse Settings" dialog.
3. Open the Microsoft Store.
4. TAB to the "See more" button and activate it to show the pop-up menu.
5. Press DOWN ARROW to go to "Downloads and updates" and activate it. The "Downloads and updates" section of the Microsoft Store will open.
6. Use the mouse to read the items in the list under the "Recent activity" header.

### Expected behavior:
NVDA should read the information displayed for each item.

### Actual behavior:
NVDA says only "WinStore.UX.Controls.Installs.UpdatedControl" for each item.

### System configuration:
- Operating system: Windows 10 Pro version 1709 (build 16299.402), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-15060,4709bfa7.
- Microsoft Store version: 11803.1001.8.0.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

### Other questions:
- The issue occurs even after restart of NVDA and with add-ons disabled.
- The issue does not occur if instead of using the mouse to read the items, the keyboard focus is moved to the list and the arrow keys are used to read them.
- The issue does not occur in the list of currently updating apps in the same view – only to the "Recent activity" list.

**Comment by Adriani90** [comment]
cc: @josephsl this is still an issue, also in Windows 10 1909 Update with NVDA 2020.1 RC1

**Comment by Adriani90** [comment]
@k-kolev1985 can you please test with NVDA 2023.1 or last alpha version? Is this still occuring on Windows 10 22 H2 update or Windows 11 22 H2 Update?

**Comment by k-kolev1985** [comment]
@Adriani90 No, with the current version of Microsoft Store and the last alpha of NVDA, this does not occur. I think it was fixed by Microsoft when they redesigned the Store app.
